### ISSUE TITLE
use variable instead of hardcoded namespace

### ISSUE THREAD

**Issue by motti24** [DNXLabs/terraform-aws-eks-grafana-prometheus#5]
### Contact Details (optional)

motti.dadison@gmail.com

### Description. What happened?

It seems that I can't use a different namespace in the grafana.yaml file.
I've fixed it by changing the file to a template and using an existing variable.  

from:
url: http://prometheus-server.prometheus.svc.cluster.local
to:
url: http://prometheus-server.${namespace_prometheus}.svc.cluster.local

in helm-grafana.tf I've changed to the following:
from:
file("${path.module}/grafana.yaml"),
to:
templatefile("${path.module}/grafana.yaml.tpl", {namespace_prometheus = var.namespace_prometheus}),

Basically, I can modify the URL in the grafana dashboard but there is a bug that no matter what value I enter it keeps trying with the hardcoded value.

### Steps to reproduce

run with a different namespace for prometheus.

### Expected behavior

to accept the correct namespace

### Actual behavior

hardcoded namespace

### Occurrence

Frequently

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
